code/src/components/ThoughtBox.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { formatDistance } from 'date-fns';
+import HeartButton from './HeartButton';
+
+const ThoughtBox = ({
+  thoughts,
+  loading,
+  setLoading,
+  fetchThoughts,
+  clickCount,
+  setClickCount
+}) => {
+  useEffect(() => {
+    setLoading(true);
+    fetchThoughts();
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
+  return (
+    <>
+      {loading && (<h3>Loading...</h3>)}
+      {!loading && thoughts.map((thought) => {
+        return (
+          // eslint-disable-next-line no-underscore-dangle
+          <div className="thought-box" key={thought._id}>
+            <p>{thought.message}</p>
+            <div className="thought-details">
+              <div className="thought-button-counter">
+                <HeartButton
+                  buttonText="&#10084;&#65039;"
+                  aria-label="heart icon"
+                  textColor="black"
+                  buttonColor="#e6e6e6"
+                  likesColor="pink"
+                  buttonWidth="40px"
+                  buttonHeight="40px"
+                  buttonRadius="50%"
+                  likes={thought.hearts}
```

Comment:
Is there a particular reason you have the color, textcolor etc for the button in here and not in the css-file? 
Nice that it works as well!